<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorRotationWidget</class>
 <widget class="QWidget" name="InspectorRotationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>168</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutRotation">
     <item>
      <spacer name="horizontalSpacerRotation">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>43</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelRotation">
       <property name="text">
        <string>Rotation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderRotation">
       <property name="minimum">
        <number>-36000</number>
       </property>
       <property name="maximum">
        <number>36000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxRotation">
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTransitionDuration">
     <item>
      <spacer name="horizontalSpacerTransitionDuration">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>41</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTransitionDuration">
       <property name="text">
        <string>Duration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxTransitionDuration">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFrm1">
       <property name="text">
        <string>frm</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTween">
     <item>
      <spacer name="horizontalSpacerTween">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>54</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTween">
       <property name="text">
        <string>Tween</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTween">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTriggerOnNext">
     <item>
      <spacer name="horizontalSpacerTriggerOnNext1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>2</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTriggerOnNext">
       <property name="text">
        <string>Trigger on next</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxTriggerOnNext">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerTriggerOnNext2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDefer">
     <item>
      <spacer name="horizontalSpacerDefer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>53</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDefer">
       <property name="text">
        <string>Defer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxDefer">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerDefer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerRotation">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBoxTransitionDuration</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorRotationWidget</receiver>
   <slot>transitionDurationChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxTween</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InspectorRotationWidget</receiver>
   <slot>tweenChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderRotation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorRotationWidget</receiver>
   <slot>sliderRotationChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxDefer</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorRotationWidget</receiver>
   <slot>deferChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTriggerOnNext</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorRotationWidget</receiver>
   <slot>triggerOnNextChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxRotation</sender>
   <signal>valueChanged(double)</signal>
   <receiver>InspectorRotationWidget</receiver>
   <slot>doubleSpinBoxRotationChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sliderRotationChanged(int)</slot>
  <slot>tweenChanged(QString)</slot>
  <slot>transitionDurationChanged(int)</slot>
  <slot>doubleSpinBoxRotationChanged(double)</slot>
  <slot>deferChanged(int)</slot>
  <slot>triggerOnNextChanged(int)</slot>
 </slots>
</ui>
